File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int res = 0;

        int left = 0;
        int right = 0;

        while(left < prices.length - 1){
            int tempRes = prices[right] - prices[left];

            System.out.println(left + " " + right + ". TempRes: " + tempRes);

            if(tempRes > res)
                res = tempRes;

            right++;

            if(right >= prices.length){
                left++;
                right = left + 1;
            }
        }

        return res;
    }
}

File: Data Structures & Algorithms/level-order-traversal-of-binary-tree/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def levelOrder(self, root: Optional[TreeNode]) -> List[List[int]]:
        res = []
        queue = collections.deque()
        queue.append(root)

        while queue:
            len_ = len(queue)
            level = []

            for i in range(len_):
                temp = queue.popleft()

                if temp:
                    queue.append(temp.left)
                    queue.append(temp.right)
                    level.append(temp.val)
            
            if level: res.append(level)

        return res
        